srcpkgs/herbstluftwm/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'herbstluftwm'
 pkgname=herbstluftwm
-version=0.9.3
+version=0.9.4
 revision=1
 build_style=cmake
 hostmakedepends="pkg-config asciidoc"
 makedepends="libXrandr-devel libXinerama-devel libglib-devel libXft-devel
-	libXrender-devel freetype-devel"
+	libXrender-devel freetype-devel libXfixes-devel"
```

Comment:
```suggestion
 libXrender-devel freetype-devel libXfixes-devel"
```
nitpick

Reply:
replace it with a space @Barbaross93

Reply:
replace it with a space @Barbaross93

Reply:
Ahh, my bad. Didn't see the space there at first.